client/state/data-layer/wpcom-http/test/index.js

```diff
@@ -77,3 +80,99 @@ describe( '#queueRequest', () => {
 		}, 10 );
 	} );
 } );
+
+describe( '#fetcherMap', () => {
+	const wpcomReq = {};
+	const noopCallback = () => {};
```

Comment:
seems awkward to name this `noopCallback` instead of just `noop`.

note also that `lodash.noop` also exists